app/Http/Controllers/Api/V1/Integration/VideoDirectUrlController.php

```diff
@@ -127,8 +134,9 @@ public function getKomodoDirectUrl($url)
     }
 
     /**
+     * Get remote content
```

Comment:
Add new line between title, description param and return.

Reply:
Done!